e2e-tests/tests/visit.spec.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
```

Comment:
My test also failed in `visit` page as well.. It looks like the snapshot test failed but not sure if this is expected as well.

![image](https://user-images.githubusercontent.com/94404633/175440068-cd2444f3-b09a-48de-b81f-8c5b024a911d.png)